### PR TITLE
flyway migration tool documentation

### LINKED ISSUE #119: Documentation in ydb.tech en / ru

### PR DESCRIPTION
**Pull request by KirillKurdyukov**
### Changelog category 

* Documentation (changelog entry is not required)

### CODE REVIEW

File: ydb/docs/en/core/integrations/flyway.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+Also, using the `repair` command, you can delete a failed DDL script.
+
+### clean {#flyway-repair}
+
+Command [clean](https://documentation.red-gate.com/flyway/flyway-cli-and-api/usage/command-line/command-line-clean) drops all objects in the configured schemas.
+
+{% note warning %}
+
+{{ ydb-short-name }} does not have a concept like `schema`. The `clean` command drops all the tables in your database.
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/flyway.md:222]:
```suggestion
Unlike other database management systems, {{ ydb-short-name }} does not have a concept of `SCHEMA`. Thus, the `clean` command **drops all the tables** in a given database. 
```

File: ydb/docs/en/core/integrations/flyway.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
+# Migrating {{ ydb-short-name }} data schemas with the Flyway migration tool
+
+## Introduction {#introduction}
+
+[Flyway](https://documentation.red-gate.com/fd/) is an open-source database migration tool. It strongly favors simplicity and convention over configuration. It has extensions for various database management systems (DBMS), including {{ ydb-short-name }}.
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/flyway.md:5]: If you follow this link, the most prominent product is Flyway Desktop. Shouldn't it be covered as well?

File: ydb/docs/en/core/integrations/flyway.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+Migration checksum mismatch for migration version 4
+-> Applied to database : 591649768
+-> Resolved locally    : 1923849782
+```
+
+### repair {#flyway-repair}
+
+Command [repair](https://documentation.red-gate.com/flyway/flyway-cli-and-api/usage/command-line/command-line-repair) tries to eliminate the identified errors and discrepancies from the database schema history table.
+
+Fix the problem with different `checksums` by running the following command:
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/flyway.md:202]:
```suggestion
Fix the problem with different `checksum`'s by running the following command:
```

**Review by blinkov — CHANGES_REQUESTED** [review]

File: ydb/docs/en/core/integrations/flyway.md (hunk 2 above)
Comment (KirillKurdyukov) [reply ydb/docs/en/core/integrations/flyway.md:5]: The Amazon CloudFront distribution is configured to block access from your country. We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.

Comment (KirillKurdyukov) [reply ydb/docs/en/core/integrations/flyway.md:5]: I can't download the desktop application, it seems to me it's an unused part.

**Review by fomichev3000 — APPROVED** [review]

**Review by blinkov — APPROVED** [review]

**Pull request merged**